functions/src/index.ts

```diff
@@ -17,60 +19,50 @@ declare global {
  * NOTE: Please do not change the function name as the firebase uses the firebase to
  * auto generate http URL i.e. base_url/region/function-name (e.g: baseurl/europe-west1/deskBooking)
  */
-exports.deskapi = functions
-  .region('europe-west1')
-  .runWith({
-    // Only allow one booking to be made at a time
-    maxInstances: 1,
-    timeoutSeconds: 60,
-  })
-  .https.onRequest(App);
-
+exports.deskapi = onRequest(
+  {region: 'europe-west1', maxInstances: 1, timeoutSeconds: 60},
```

Comment:
Wondering since a lot of the fucntions use the same region, timeout and max instances if consts of those values could be made and then referenced instead since there is repetition

Reply:
Added `const config = {region: 'europe-west1', maxInstances: 1, timeoutSeconds: 60};`